### DISCUSSION TITLE
One-hot encoding y columna con identificadores

### DISCUSSION THREAD

**Discussion by HM9N** [rramosp/ai4eng.v1#253] [UDEA Students]
Buenos días

He aplicado one-hot encoding al dataset de mi proyecto para convertir las variables categóricas a numéricas. Sin embargo, tengo dudas respecto al gran incremento que obtuve en el número de columnas. En un inicio tenía 78 columnas y luego de aplicar one-hot obtuve 2953 columnas. Mi duda es cómo saber si voy por buen camino o si apliqué bien la técnica. 

Otra duda que tengo es respecto a una columna que tiene identificadores y por esto, todos los valores son diferentes. ¿Esta columna se elimina?

Gracias

**Comment by rramosp** [comment]
hola john, ninguna aproximación es a priori mejor o peor ... toca hacer la
experimentación (realizar predicciones y medir el desempeño) y comparar
cada opción.

sí, las columnas de identificadores se eliminan, no proporcionan información

On Wed, Apr 26, 2023 at 7:47 AM Jhon ***@***.***> wrote:

> Buenos días
>
> He aplicado one-hot encoding al dataset de mi proyecto para convertir las
> variables categóricas a numéricas. Sin embargo, tengo dudas respecto al
> gran incremento que obtuve en el número de columnas. En un inicio tenía 78
> columnas y luego de aplicar one-hot obtuve 2953 columnas. Mi duda es cómo
> saber si voy por buen camino o si apliqué bien la técnica.
>
> Otra duda que tengo es respecto a una columna que tiene identificadores y
> por esto, todos los valores son diferentes. ¿Esta columna se elimina?
>
> Gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/253>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOPB5XBUQXPTR4ARSLXDEKOBANCNFSM6AAAAAAXMMMT5Y>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
